tutorials/using-scheduler-invoke-private-functions-oidc/index.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Using Cloud Scheduler to invoke private Cloud Functions with OIDC
+description: Learn how to use OpenID Connect to invoke private Cloud Functions periodically with Cloud Scheduler. No Pub/Sub required!
```

Comment:
The tests REGEX is expecting the phrase to end with a `.`    Maybe it should be "Bypass Pub/Sub and learn how to use OpenID Connect to ..."  Or maybe keep it as is and replace the `!` with a period:   "No Pub/Sub required."

Reply:
I fixed the description, made the title match our style, and removed the tags that are also in the title.